src/Formatter/Pool.php

```diff
@@ -14,9 +14,9 @@
 use Psr\Log\LoggerAwareInterface;
 use Psr\Log\LoggerInterface;
 use Psr\Log\NullLogger;
-use Twig_Environment;
-use Twig_Error_Syntax;
-use Twig_Sandbox_SecurityError;
+use Twig\Environment;
+use Twig\Error\SyntaxError as TwigSyntaxError;
```

Comment:
Why did you use an alias?

Reply:
Hmmm I don't know why rector did this, let's remove it.